pkg/openfeature/client.go

```diff
@@ -5,6 +5,10 @@ import (
 	"fmt"
 )
 
+const (
+	TargetingKey string = "targetingKey" // flattened evaluation context key for the TargetingKey field
```

Comment:
I find this comment a bit confusing, Since it's exported we might want to explain (very high level) what the `TargetingKey` is.

Reply:
I've updated this comment, pulled a line from the spec - let me know if it is suitable